editor/src/core/es-modules/test-cases/file-with-es-module.json

```diff
@@ -1,7 +1,7 @@
 {
   "contents": {
     "/node_modules/mypackage/index.js": {
-      "content": "export const cica = 'hello';",
+      "content": "export const hello = 'hello!';",
```

Comment:
awwww

Reply:
Just made it more consistent with another file that looks the same. :)